pkg/create/templates/test.go

```diff
@@ -43,7 +43,19 @@ container_exists() {
 }
 
 container_ip() {
-  docker inspect --format="{{"{{"}} .NetworkSettings.IPAddress {{"}}"}}" $(cat $cid_file)
+  if (echo "$OSTYPE" | grep -qs 'darwin'); then
```

Comment:
True ... the problem I see with docker-machine is that it may be installed and it might even have a valid ip but it could be that I'm configured to point to a different Docker. For example, in my case, I have both docker-machine and Docker for Mac ... docker-machine ip will return a valid ip because I do have a default machine created. However, most of the time my DOCKER_HOST variable is not set, which means that 'docker run' will use Docker for Mac.